diagnostic.md

```diff
@@ -35,7 +35,9 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    The first route definition actually specifies two routes: products and products/:product_id. The second only specifies one: products/:product_id.
```

Comment:
Can you expand on that?  These two route definitions are identical  URL paths.
One of the biggest distinctions is where Ember will render the content.  The nested `'products'/'product'` instructs Ember to render `app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.